packages/menu/src/Menu/Menu.tsx

```diff
@@ -231,10 +231,6 @@ export const Menu = React.forwardRef<HTMLDivElement, MenuProps>(function Menu(
           return React.cloneElement(child, {
             ref: setRef,
             onFocus,
-            onClick: (e: React.MouseEvent) => {
-              child.props?.onClick?.(e);
-              handleClose();
-            },
```

Comment:
this is now removing the custom `onClick` passed to each menu item. do we not still want to retain that functionality?

Reply:
we have tests testing that the callback fires on click

Reply:
oh I see I guess it still is as part of the `React.cloneElement` function!